jenkins/scripts/pulse_in_pipeline_scanning/utils/es.py

```
@@ -137,11 +77,77 @@ def get_latest_license_preapproved_container_deps(scan_type: str):
+    if container:
+        filters.append({"term": {"s_release_image": container}})
+    try:
+        resp = ES_CLIENT.search(
+            index=ES_INDEX_BASE + "-*",
+            body={
+                "size": 10000,
+                "query": {"bool": {"filter": filters}},
+                "_source": ["s_package_name", "s_ticket_url"],
+            },
+        )
+    except Exception as exc:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)